### PR TITLE
Image feedback fix

### PR DESCRIPTION
**Pull request by LorenzMeier**

### CODE REVIEW

File: src/gazebo_geotagged_images_plugin.cpp [hunk 1]
```diff
@@ -201,10 +202,12 @@ void GeotaggedImagesPlugin::OnUpdate(const common::UpdateInfo& /*_info*/) {
...
 
-void GeotaggedImagesPlugin::OnNewGpsPosition(ConstVector3dPtr& v)
-{
-  lastGpsPosition_ = *v;
-  //gzdbg << "got gps pos: "<<lastGpsPosition_.x()<<", "<<lastGpsPosition.y()<<endl;
+void GeotaggedImagesPlugin::OnNewGpsPosition(GpsPtr& gps_msg) {
+  lastGpsPosition_.x = gps_msg->latitude_deg();
+  lastGpsPosition_.y = gps_msg->longitude_deg();
+  lastGpsPosition_.z = gps_msg->altitude();
+  gzerr << "NEW GPS POS";
```
Comment (LorenzMeier) [inline src/gazebo_geotagged_images_plugin.cpp]: @bkueng This gets never called - do you see why this could be?

Comment (bkueng) [reply src/gazebo_geotagged_images_plugin.cpp]: I got it to work when I initialize the node handle without providing the namespace: `node_handle_->Init();` I'm not sure what the 'proper' way is though.

Comment (bkueng) [reply src/gazebo_geotagged_images_plugin.cpp]: I reworked the commit & removed the debug printf's

**Comment by bkueng** [comment]
I see another issue:
```
Warning [parser.cc:838] XML Element[robotNamespace], child of element[model] not defined in SDF. Ignoring[robotNamespace]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
```
This is with gazebo 8.1.1.

**Comment by LorenzMeier** [comment]
Would probably need to be removed in the offending section.

**Pull request merged**